src/reducers/__tests__/deleteThought.ts

```diff
@@ -80,3 +80,20 @@ it('update cursor after thought deletion', () => {
     },
   ])
 })
+it('deletion of empty thought', () => {
```

Comment:
The test case is not about deleting any empty thought, but about deleting a second empty thought. Let's update the test description to be more specific. 

`it('delete the correct empty thought when there are multiple', ...)`